pkg/project/version_bundle_kvm.go

```diff
@@ -22,7 +22,7 @@ var versionBundleKVM = versionbundle.Bundle{
 		},
 		{
 			Name:    "nginx-ingress-controller",
-			Version: "0.27.1",
+			Version: "0.28.0",
```

Comment:
@MarcelMue considering that the change is bump of nginx-ingress-controller, should this go into patch 0.22.1 or start next legacy cluster-operator minor 0.23.0?

Reply:
Let's do a minor.